scripts/pylib/twister/twisterlib/twister_main.py

```diff
@@ -114,7 +114,7 @@ def twister(options: argparse.Namespace, default_options: argparse.Namespace):
             if i.status == TwisterStatus.FILTER:
                 if options.platform and not tplan.check_platform(i.platform, options.platform):
                     continue
-                logger.debug(
+                logger.info(
```

Comment:
This is very verbose, even in debug mode, given that the comment field is pretty much freestyle and someone might end up writing a novel there. Maybe we should have a field for an issue number, so it is managed and can be added to the console output, right now:

![image](https://github.com/user-attachments/assets/f67f825c-ebb5-4ccc-b817-3987183fb2cd)

Reply:
btw now we already will need some differentiation between "quarantine" skips and "other" skips, (not showing the whole reason in logs). I like the idea of some extra formatting field for issue number, but please keep in mind that it will add even more tricky handling of skip types at reporting level. that's why I preferred to have a dedicated status for quarantined items.